apps/storefront/pages/grunnleggende/designelementer/dynamisk-typografi.mdx

```diff
@@ -0,0 +1,72 @@
+import { Figma } from 'mdx-embed';
+import { TokenIcon } from '@navikt/aksel-icons';
+
+import { MenuPageLayout } from '@layouts';
+import { Image, Meta } from '@components';
+
+<Meta
+  title='Dynamisk typografi'
+  description='Det er mulig å bruke dynamiske font-størrelser uten breakpoints'
+/>
+
+export default ({ children }) => (
+  <MenuPageLayout
+    content={children}
+    data={{
+      title: 'Dynamisk typografi',
+      date: '28. april 2024',
+      icon: <TokenIcon />,
+      color: 'blue',
+    }}
+  />
+);
+
+Ved hjelp av “Fluid typography” kan vi skalere tekststørrelsen gradvis mellom en min og maks-bredde, i stedet for at det blir en brå oppskalering på et spesifikt breakpoint. Den lineære funksjonen tilnærmer seg ønsket størrelse jevnt over skjermens bredde.
+
+Tekststørrelsene øker gradvis fra viewport-bredde 320px helt til det treffer maksbredden på viewporten som er satt til 1360px. En tekst på 16px vil f.eks øke til 18px på den største viewporten.
+
+<Image
+  src='/img/fluid.png'
+  alt='Illustrasjonen viser at en 16px tekststørrelser på små skjermer øker gradvis opp til 18px på store skjermer.'
+  boxShadow={false}
+/>
+
+## Fordeler
+
+- Du slipper å tenke på breakpoints, størrelsen på skriften blir automatisk tilpasset skjermen.
+- Når størrelsene ikke justeres på konkrete breakpoints unngår vi at det blir stykkevis og brå oppskaleringer.
+
+## Ulemper
+
+- Halve piksler. Figma runder nedover. Noen nettlesere runder oppover, andre nedover. Det gjør at ting ikke ser likt ut i Figma og i nettleseren.
+- Du kan føle du har mindre kontroll.
+- I Figma kan denne størrelsesendringen kun skje ved bruk av pluginen "Token Studio", når du har aktivert ønsket viewport for din frame.
+- Det er ikke gitt at et produkt vil bygge sitt produkt ved hjelp av dynamisk typografi. Dersom noen legger til moduler fra designsystemet i eget grensesnitt vil det ikke samsvare med skaleringen deres.
+
+<br />
+
+**Basert på [ulempene dette medførte](https://github.com/digdir/designsystemet/issues/1444), valgte vi å gå for en [statisk typografi-skala](/grunnleggende/designelementer/typografi#tekststørrelse) som standard for designsystemet.**
+
+## Du kan fortsatt velge å bruke fluid
+
+Du kan fortsatt velge å slå på dynamisk typografi, ved å aktivere settet "[fluid](https://github.com/digdir/designsystemet/blob/main/design-tokens/Base/Fluid.json)".
```

Comment:
De må vel og definere riktig viewport på framen de jobber i for at det skal fungere? De settene har vi vel kanskje fjernet helt 🤔